build/ci.go

```diff
@@ -317,6 +318,27 @@ func downloadSpecTestFixtures(csdb *build.ChecksumDB, cachedir string) string {
 	return filepath.Join(cachedir, base)
 }
 
+// downloadPragueSpecTestFixtures downloads and extracts the execution-spec-tests fixtures for Prague.
+func downloadPragueSpecTestFixtures(csdb *build.ChecksumDB, cachedir string) string {
```

Comment:
Can we reuse `downloadSpecTestFixtures`?

Reply:
It may be possible but, this gonna be remove after Prague is released.